gcloud-java-storage/src/main/java/com/google/gcloud/storage/Blob.java

```diff
@@ -256,4 +261,88 @@ public URL signUrl(long expirationTimeInSeconds, SignUrlOption... options) {
   public Storage storage() {
     return storage;
   }
+
+  /**
+   * Gets the requested blobs. If {@code infos.length == 0} an empty list is returned. If
```

Comment:
You are right. We'll have to use `Collections.unmodifiableList` as guava's `ImmutableList` does not support `null` as a value, right?